Mage/src/main/java/mage/abilities/dynamicvalue/common/GetMonstrosityXValue.java

```diff
@@ -0,0 +1,37 @@
+package mage.abilities.dynamicvalue.common;
+
+import mage.abilities.Ability;
+import mage.abilities.common.BecomesMonstrousSourceTriggeredAbility;
+import mage.abilities.dynamicvalue.DynamicValue;
+import mage.abilities.effects.Effect;
+import mage.game.Game;
+
+
+/**
+ * Monstrosity X
+ *
+ * @author notgreat
+ */
+public enum GetMonstrosityXValue implements DynamicValue {
+    instance;
+
+    @Override
+    public int calculate(Game game, Ability sourceAbility, Effect effect) {
+        return ((BecomesMonstrousSourceTriggeredAbility) sourceAbility).getMonstrosityValue();
```

Comment:
Do not use such code style in public classes. Must put class back to ability file or add instanceof check and raise error on wrong code usage instead.